documentation/Versioning.md

```diff
@@ -0,0 +1,9 @@
+# Versioning
+
+To make sure we don't mess up data on the production server, there are three levels of versioning:
+
+1. Development. Operates on the developers computer only. Uses the `space-edventures-beta` Firebase app and the test Stripe API. Signified by the `NODE_ENV` environment variable not being 'production' and the `REACT_APP_IS_LIVE` environment variable not being present.
+
```

Comment:
Do I need to set the NODE_ENV in my .env file? Or is that part of something else?